source/blog/2015-12-08-fedora-atomic-sssd-container.html.md

```diff
@@ -0,0 +1,90 @@
+---
+title: External authentication for Fedora Atomic Host
+author: adelton
+date: 2015-12-08 16:00:00 UTC
+tags: fedora, atomic host, sssd, authentication
+comments: true
+published: true
+---
+
+By default, Fedora Atomic images come with cloud-init which supports customization of various aspects of the running instance, including password for the default user. However, if many users in large organization should have access to the Atomic Host, the local configuration of the instance will not scale and user identities, authentication, and authorization need to be based on external identity management solution. For IPA (FreeIPA/IdM), Active Directory, or generic LDAP servers, SSSD can serve as an agent providing these services, from user identity lookups and user group membership resolution to access control. With SSSD container now available, Fedora Atomic Hosts can be deployed in very similar way to normal Fedora.
```

Comment:
"With SSSD container now available," should that be "an SSSD container" or "the SSSD container"?